doc/neps/conf.py

```diff
@@ -33,6 +33,18 @@
 extensions = [
     'sphinx.ext.imgmath',
     'sphinx.ext.intersphinx',
+    'sphinx_affiliates',
+]
+
+
+# Canonical URL (including the '/') so searching from numpy docs finds these
+affiliate_options = {
+    'canonical_url': "https://www.numpy.org/neps/",
+}
+
+# Other sites to add to the search of this site
+sphinx_affiliates = [
+    'https://www.numpy.org/devdocs/affiliate_searchindex.js',
```

Comment:
Are we sure we want devdocs and not stable?

Reply:
I guess it depends who the audience is for search from the NEP site back to the main site: is it people researching how something was done, or people looking for how some new feature, not yet released is documented.

In any case if it is decided to use the stable docs for reference we can create the `affiliate_searchindex.js` file manually by editing the `https://www.numpy.org/doc/stable/searchindex.js` one and committing it.